pkg/comp-functions/runtime/function_mgr.go

```diff
@@ -482,6 +482,14 @@ func KubeOptionAddRefs(refs ...xkube.Reference) KubeObjectOption {
 	}
 }
 
+// KubeOptionAllowDeletion adds the allow deletion label to the object
+func KubeOptionAllowDeletion(obj *xkube.Object) {
+	if obj.ObjectMeta.Labels == nil {
+		obj.ObjectMeta.Labels = map[string]string{}
+	}
+	obj.ObjectMeta.Labels[WebhookAllowDeletionLabel] = "true"
+}
+
```

Comment:
why do we have this here? I can't find references in code

Reply:
It's used multiple times in `proxysql.go`:
https://github.com/vshn/appcat/pull/249/files#diff-6c00f19154044c901c94120e2e51fa40f7ccb6e422335f9172257d054447d96dR198
https://github.com/vshn/appcat/pull/249/files#diff-6c00f19154044c901c94120e2e51fa40f7ccb6e422335f9172257d054447d96dR258
https://github.com/vshn/appcat/pull/249/files#diff-6c00f19154044c901c94120e2e51fa40f7ccb6e422335f9172257d054447d96dR414
https://github.com/vshn/appcat/pull/249/files#diff-6c00f19154044c901c94120e2e51fa40f7ccb6e422335f9172257d054447d96dR476
https://github.com/vshn/appcat/pull/249/files#diff-6c00f19154044c901c94120e2e51fa40f7ccb6e422335f9172257d054447d96dR501

Reply:
file was hidden :see_no_evil: now it makes more sense :D